extensions/standard-processors/processors/HashContent.cpp

```diff
@@ -84,6 +82,7 @@ void HashContent::onTrigger(core::ProcessContext *, core::ProcessSession *sessio
   }
 
   if (failOnEmpty_ && flowFile->getSize() == 0) {
+    logger_->log_trace("Failure as flow file is empty");
```

Comment:
Minor, but trace level logging seems misused here, most things loggged in this class seem clearly useful for debugging.

Reply:
Yeah I added it to checks logs in testing. How do you suggest I tweak this?

Reply:
Just change this to at least log_debug.

Reply:
Okay. Done :)